src/ontology/slurp/ordo.tsv

```diff
@@ -1,13 +1,13 @@
-MONDO:0958339	syndromic genetic disorder with strabismus	Orphanet:522520	MONDO:equivalentTo	Syndromic genetic disorder with strabismus		MONDO:8000033
-MONDO:0958340	rare genetic disorder with entropion	Orphanet:522530	MONDO:equivalentTo	Rare genetic disorder with entropion		MONDO:8000033
-MONDO:0958341	genetic superficial corneal dystrophy	Orphanet:522562	MONDO:equivalentTo	Genetic superficial corneal dystrophy		MONDO:8000033
-MONDO:0968986	chaotic conus spinal cord lipoma	Orphanet:645285	MONDO:equivalentTo	Chaotic conus spinal cord lipoma	A rare dysraphic spinal cord lipoma characterized by the lipomatous mass extending ventrally to the dorsal root entry zone, indicating a more severe malformation of the spinal cord. The diagnosis can be suggested on imaging but usually confirmed during surgery.	MONDO:0958350|MONDO:8000034|MONDO:8000030
-MONDO:0968987	extramedullary conus spinal cord lipoma	Orphanet:645297	MONDO:equivalentTo	Extramedullary conus spinal cord lipoma	A rare closed lipomatous, dysraphic malformation of the lower spinal cord characterized by extramedullary lipomatous mass attached to the conus region. The conus is dysplastic and poorly delineated. Various morphological subtypes are recognized. Possible symptoms include bowel and bladder dysfunction and neuro-orthopedic deformity of the lower limbs.	MONDO:0958350|MONDO:8000034|MONDO:8000030
-MONDO:0968988	saccular spinal dysraphism with a stalk to the dome	Orphanet:645319	MONDO:equivalentTo	Saccular spinal dysraphism with a stalk to the dome	A rare spinal dysraphism characterized by a meningocele, containing a stalk, that is attached to the inner surface of the meningocele. The stalk can be posteriorly fibroneural (saccular limited dorsal myeloschisis) or the spinal cord itself (myelic limited dorsal malformation).	MONDO:0958345|MONDO:8000033|MONDO:0017069
-MONDO:0968989	non-saccular limited dorsal myeloschisis	Orphanet:645343	MONDO:equivalentTo	Non-saccular limited dorsal myeloschisis	A rare form of limited dorsal myeloschisis (LDM), characterized by a non saccular cutaneous stigmata (midline skin abnormality classically dimple, pit or sometimes angioma), the stalk is attached to this cutaneous stigmata. Fibroneural stalk varies in thickness and complexity.	MONDO:0958345|MONDO:8000030|MONDO:8000034
-MONDO:0968990	genetic central precocious puberty in male	Orphanet:650097	MONDO:equivalentTo	Genetic central precocious puberty in male		MONDO:8000031|MONDO:0958356
-MONDO:0968991	non-genetic central precocious puberty in male	Orphanet:650102	MONDO:equivalentTo	Non-genetic central precocious puberty in male		MONDO:0958356|MONDO:8000031
+mondo_id	mondo_label	xref	xref_source	original_label	definition	parents	subset
+ID	LABEL	A oboInOwl:hasDbXref	>A oboInOwl:source SPLIT=|		A IAO:0000115	SC %	
+MONDO:0958337	disorder with optic nerve compression	Orphanet:519337	MONDO:equivalentTo	Disorder with optic nerve compression		MONDO:8000033	group of disorders
```

Comment:
## Output
@sabrinatoro Here you are! Check out this [google sheet](https://docs.google.com/spreadsheets/d/13styiaK6DhF03VF3IEGg-78sJH1pICOfPWzG5iowUpY/edit#gid=1585402401).

Reply:
@sabrinatoro You don't need this merged into `main` any time soon, do you? Or, phrased another way--do you foresee needing this information to show up in another build very soon?

If not, then I suppose that the google sheet will be fine for your immediate purposes. I only bring this up because I was planning to merge this into `develop`, which is our typical workflow,  but I don't know when the next time is that `develop` would get merged into `main`. If you foresee that you'll need to see this information again in a build very soon, then I think we can do a merge of `develop` into `main` after this PR is merged.

Reply:
@sabrinatoro Does the `subset` column look good to you?
It will show one of:
- disorder
- group of disorders
- subtype of a disorder

If you want, alternatively I could have it show one of these things instead:
```python
    'Orphanet:557493': 'http://purl.obolibrary.org/obo/mondo#ordo_disease',  # disorder
    'Orphanet:557492': 'http://purl.obolibrary.org/obo/mondo#ordo_group_of_disorders',  # group of disorders
    'Orphanet:557494': 'http://purl.obolibrary.org/obo/mondo#ordo_clinical_subtype',  # subtype of a disorder
```

Reply:
I also took a look at the spreadsheet and things look good

Joe, if in the future you're interested to know more about Orphanet for things like this, one place to check is the Orphanet web site (https://www.orpha.net/en/disease) and search for a disease, e.g. saccular spinal dysraphism with a stalk to the dome. When you click on the resulting [link](https://www.orpha.net/en/disease/detail/645319?name=saccular%20spinal%20dysraphism%20with%20a%20stalk%20to%20the%20dome&mode=name) for the disease name and scroll down the Classification level is listed on the page.

Reply:
[Sabrina approves](https://github.com/monarch-initiative/mondo-ingest/pull/531#issuecomment-2128148710)! Marking this one resolved.